### PR TITLE
New Relationships Viewer with Filtering

### PR DESCRIPTION
**Pull request by ghost**
try on `3ddemo` or `akerbp`

image

### CODE REVIEW

File: src/containers/NetworkViewers/RelationshipTreeViewer.tsx [hunk 1]
```diff
@@ -115,32 +194,146 @@ class TreeViewer extends Component<Props, State> {
...
+        const ids = node.resourceId.split(':');
+        const model = models[Number(ids[ids.length - 1])];
+        if (ids.length === 3) {
+          return `Node in ${model ? model.name : 'Loading...'}`;
+        }
+        return `Linked to ${model ? model.name : 'Loading...'}`;
+      }
+      case 'timeSeries': {
+        const timeseries = timeseriesData[Number(node.resourceId)];
+        console.log(timeseries);
```
Comment (anderscognite) [inline src/containers/NetworkViewers/RelationshipTreeViewer.tsx]:
```suggestion
```

File: src/containers/NetworkViewers/RelationshipTreeViewer.tsx [hunk 2]
```diff
@@ -150,12 +343,86 @@ class TreeViewer extends Component<Props, State> {
...
+      Asset: { resource: 'asset', resourceId: '-1' },
+      Timeseries: { resource: 'timeSeries', resourceId: '-1' },
+      '3D': { resource: 'threeD', resourceId: '-1:-1' },
+    };
+    const relationships: { [key: string]: Relationship } = {
+      'Flows To': {
+        relationshipType: 'flowsTo',
+        source: nodes.Asset,
+        target: nodes.Asset,
+        confidence: 1,
```
Comment (anderscognite) [inline src/containers/NetworkViewers/RelationshipTreeViewer.tsx]: Why hard coded to 1?

File: src/containers/NetworkViewers/RelationshipTreeViewer.tsx [hunk 3]
```diff
@@ -165,25 +432,49 @@ class TreeViewer extends Component<Props, State> {
           dagMode={controls === 'none' ? null : controls}
           dagLevelDistance={300}
           backgroundColor={BGCOLOR}
-          linkColor={() => 'rgba(255,255,255,0.2)'}
+          linkColor={this.chooseRelationshipColor}
```
Comment (anderscognite) [inline src/containers/NetworkViewers/RelationshipTreeViewer.tsx]: `linkColor` seems to be a color, not a function. Should it be `linkColorFunc`?

**Review by anderscognite — APPROVED** [review]
See my comments/questions!

File: src/containers/NetworkViewers/RelationshipTreeViewer.tsx (hunk 2 above)
Comment (dlcognite) [reply src/containers/NetworkViewers/RelationshipTreeViewer.tsx]: because in this case it doesnt matter :P just for generating the legend cleanly

**Review by anderscognite — APPROVED** [review]

**Pull request merged**